packages/hint-manifest-icons/tests/tests.ts

```diff
@@ -214,7 +214,123 @@ const tests: HintTest[] = [
                                 "sizes": "192x192",
                                 "type": "image/png"
                             }
-                            ]
+                        ]
+                    }`
+            }
+        }
+    },
+    {
+        name: 'Duplicated purpose found',
```

Comment:
sorry for being so pedantic
```suggestion
        name: 'Duplicate purpose found',
```

Reply:
🤦‍♂️I forgot to save the change before pushing... 
![image](https://user-images.githubusercontent.com/1581288/86068031-2c6e2f00-ba2b-11ea-983f-b810f580c10a.png)

Reply:
English isn't @sarvaje's nor mine first language so any grammar, spelling, or style corrections are more than welcome 😊

Thanks a lot!

Reply:
Thanks for the correction!!, as @molant says, English isn't our first language :)

Reply:
english isn't my first language either lol. the correction was more about consistency, as `duplicate` was used in other cases, not `duplicated`. (plus `duplicate` does sound better in my ears, but english is my second language so i'm not that reliable.)